common/app/views/support/Profile.scala

```diff
@@ -24,7 +24,7 @@ sealed trait ElementProfile {
     maybeAsset.flatMap(_.url).map(ImgSrc(_, this))
 
   def bestFor(image: ImageMedia): Option[ImageAsset] = {
-    if(ImageServerSwitch.isSwitchedOn) {
+    if(!ImageServerSwitch.isSwitchedOn) {
```

Comment:
Agreed, I thought the same thing, I am planning for this logic to be replaced soon anyway.